js/app/modules/card/list.js

```diff
@@ -30,6 +30,21 @@ const List = new Module.Class({
             true),
     },
 
+
+    StyleProperties: {
+        /**
+         * This property determines how we scale the thumbnail image. It uses a
+         * similar vocabulary as web css' background-size property: 'cover' and
+         * 'center'. 'cover' means we will reposition and rescale the image to
+         * cover the entire thumbnail space. 'center' means we will maintain the
+         * image's original dimensions and center it in the thumbnail space.
+         */
+        'thumbnail-background-size': GObject.ParamSpec.string('thumbnail-background-size',
```

Comment:
Adding something as a CSS property means we have to be prepared for someone to do something preposterous like change it to a different value on hover. I'm thinking for this case it might make more sense to have a construct-only property even though it does feel like more of a theming thing.